src/SFML/Audio/SoundFileReaderFlac.cpp

```diff
@@ -142,10 +138,12 @@ namespace
                         break;
                 }
 
+                // If there's room in the output buffer, copy the sample there
                 if (data->remaining > 0)
                 {
-                    // There's room in the output buffer, copy the sample there
-                    *data->buffer++ = sample;
+                    // If there's no output buffer, it means that we are seeking, so just skip the output write
+                    if (data->buffer)
+                        *data->buffer++ = sample;
                     data->remaining--;
```

Comment:
What's the effect of decrementing data->remaining without adding data to the buffer? Well, is it useful at all to execute this statement if there's no buffer?